apis/openstack-nova/src/test/java/org/jclouds/openstack/nova/v2_0/compute/NovaComputeServiceLiveTest.java

```diff
@@ -16,130 +16,31 @@
  */
 package org.jclouds.openstack.nova.v2_0.compute;
 
-import static com.google.common.base.Charsets.UTF_8;
 import static java.util.logging.Logger.getAnonymousLogger;
 
-import java.io.File;
-import java.io.IOException;
 import java.util.Properties;
 
-import com.google.common.collect.ImmutableSet;
-import com.google.common.io.Files;
-import com.google.inject.Module;
-import org.jclouds.Context;
-import org.jclouds.ContextBuilder;
-import org.jclouds.compute.domain.TemplateBuilder;
 import org.jclouds.compute.internal.BaseComputeServiceLiveTest;
-import org.jclouds.config.ContextLinking;
-import org.jclouds.encryption.bouncycastle.config.BouncyCastleCryptoModule;
-import org.jclouds.logging.config.LoggingModule;
-import org.jclouds.logging.slf4j.config.SLF4JLoggingModule;
 import org.jclouds.openstack.keystone.config.KeystoneProperties;
-import org.jclouds.openstack.nova.v2_0.compute.options.NovaTemplateOptions;
 import org.jclouds.openstack.nova.v2_0.config.NovaProperties;
-import org.jclouds.openstack.nova.v2_0.domain.BlockDeviceMapping;
 import org.jclouds.rest.AuthorizationException;
 import org.jclouds.sshj.config.SshjSshClientModule;
 import org.testng.annotations.Test;
 
+import com.google.inject.Module;
+
 @Test(groups = "live", singleThreaded = true, testName = "NovaComputeServiceLiveTest")
 public class NovaComputeServiceLiveTest extends BaseComputeServiceLiveTest {
 
-   private Context neutronApiContext;
-
    public NovaComputeServiceLiveTest() {
       provider = "openstack-nova";
-
-      Properties overrides = setupProperties();
-      neutronApiContext = ContextBuilder.newBuilder("openstack-neutron")
-              .endpoint(setIfTestSystemPropertyPresent(overrides,
-                      "openstack-nova.endpoint"))
-              .credentials(setIfTestSystemPropertyPresent(overrides,
-                      "openstack-nova.identity"),
-                      setIfTestSystemPropertyPresent(overrides, "openstack-nova.credential"))
-              .modules(ImmutableSet.<Module>of(
-                      new SshjSshClientModule(),
-                      new SLF4JLoggingModule(),
-                      new BouncyCastleCryptoModule())
-              )
-              //        TypeToken<? extends Context> contextType = new TypeToken<ApiContext<NeutronApi>>() {};
-              .build();
```

Comment:
Now there is no way to test the neutron integration? We should provide a NovaWithNeutron live test class to test this new feature.

Reply:
ok I'll add it